### PR TITLE
Feature/price changes

### PR DESCRIPTION
**Pull request by Hicks224**
All prices for ores, wood, gathered crafting materials, raw food ingredients, and creature loot table items have been set according to the following table set by Kanlaeel and okay'd by Zunath.

<html>
<body>
<google-sheets-html-origin>

  | Ore | Refined/Wood | Gathered | Loot
-- | -- | -- | -- | --
T1 | 3 | 8 | 5 | 50
T2 | 6 | 16 | 10 | 100
T3 | 9 | 24 | 15 | 150
T4 | 12 | 32 | 20 | 200
T5 | 15 | 40 | 25 | 250

</body>
</html>

Gems and space equivalents have been set at 100 per tier.
Crafted items have not been touched.

All prices have been set to 2.5x in the files to reflect the 0.4x modifier that vendors buy items at.

### CODE REVIEW

File: Module/uti/abdamaryllia.uti.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "__data_type": "UTI ",
   "AddCost": {
     "type": "dword",
-    "value": 25
+    "value": 250
```
Comment (zunath) [inline Module/uti/abdamaryllia.uti.json:5]: Abdamaryllia isn't a tier 5 item so I'm confused why it's priced as one based on your chart? Can you explain what I'm missing?

File: Module/uti/agent_belt.uti.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "__data_type": "UTI ",
   "AddCost": {
     "type": "dword",
-    "value": 0
+    "value": 375
```
Comment (zunath) [inline Module/uti/agent_belt.uti.json:5]: Agent belts are dropped on Hutlar which is a level 30-35. The pricing should be closer aligned to the T3/T4 pricing for standard items. It's currently worth over double of those items. T3 is 63cr, T4 is 157cr. A number somewhere in this range seems more applicable to me.

File: Module/uti/byysk_armor.uti.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "__data_type": "UTI ",
   "AddCost": {
     "type": "dword",
-    "value": 0
+    "value": 375
```
Comment (zunath) [inline Module/uti/byysk_armor.uti.json:5]: Same comment about agent gear applies here too. It looks like a static 375 was set but please ensure the numbers are within the T3/T4 range.

**Review by zunath — CHANGES_REQUESTED** [review]
Requesting changes/answers to comments above.

File: Module/uti/cairnmog_blood.uti.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "__data_type": "UTI ",
   "AddCost": {
     "type": "dword",
-    "value": 26
+    "value": 125
```
[the remark this answers is not in the document]
Comment (Hicks224) [reply Module/uti/cairnmog_blood.uti.json:5]: "All prices have been set to 2.5x in the files to reflect the 0.4x modifier that vendors buy items at."

This means that 40% of 125 is 50, which is the correct buying price. IE: the player will only receive 50 credits for selling this item.

File: Module/uti/agent_belt.uti.json (hunk 2 above)
Comment (Hicks224) [reply Module/uti/agent_belt.uti.json:5]: "All prices have been set to 2.5x in the files to reflect the 0.4x modifier that vendors buy items at."

40% of 375 is 150, which is the correct buying price for a tier 3 loot item according to the table above that you okay'd. IE: the player will only receive 150 credits for selling this item.

File: Module/uti/byysk_armor.uti.json (hunk 3 above)
Comment (Hicks224) [reply Module/uti/byysk_armor.uti.json:5]: "All prices have been set to 2.5x in the files to reflect the 0.4x modifier that vendors buy items at."

40% of 375 is 150, which is the correct buying price for a tier 3 loot item according to the table above that you okay'd. IE: the player will only receive 150 credits for selling this item.

File: Module/uti/abdamaryllia.uti.json (hunk 1 above)
Comment (Hicks224) [reply Module/uti/abdamaryllia.uti.json:5]: "All prices have been set to 2.5x in the files to reflect the 0.4x modifier that vendors buy items at."

40% of 250 is 100, which is the correct buying price for a tier 2 loot item according to the table above that you okay'd. IE: the player will only receive 100 credits for selling this item.

Comment (zunath) [reply Module/uti/abdamaryllia.uti.json:5]: The 40% was what I was missing. Thanks

**Review by zunath — APPROVED** [review]

**Pull request merged**